XenAdmin/SettingsPanels/ClusteringEditPage.cs

```diff
@@ -82,7 +82,7 @@ public void SetXenObjects(IXenObject orig, IXenObject clone)
         {
             pool = Helpers.GetPoolOfOne(clone.Connection);
             var master = Helpers.GetMaster(clone.Connection);
-            flowLayoutInfo.Visible = false;
+            tableLayoutInfo.Visible = false;

```

Comment:
This panel nees better anchoring, because if the warning text is too long (if not in english, perhaps in l10n) resizing will not happen correctly.
There is also something wrong with the top lable as well (or its parent controls) because resizing the properties dialog causes text truncation.